### PR TITLE
Add more detail to overlay survey instructions

### PR DESCRIPTION
**Pull request by bboston7**
This change adds some additional instructions to the overlay survey section of the admin guide. Specifically, it adds:

* Instructions on how to use the overlay survey script.
* Information about the `nonce` field in the "start/stop collecting" messages.
* Explicit instructions on how to opt-out of the survey.
* Recommendations for collecting phase durations.
* Small tweaks to fix grammar/formatting/example errors found while editing the section

### CODE REVIEW

File: network/core-node/admin-guide/monitoring.mdx
Comment (Julian-dev28) [inline network/core-node/admin-guide/monitoring.mdx:187]: please add links to the scripts directory

File: network/core-node/admin-guide/monitoring.mdx
Comment (Julian-dev28) [inline network/core-node/admin-guide/monitoring.mdx:190]: I'm running into issues with running this script. 

```
requests.exceptions.ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=11626): Max retries exceeded with url: /startsurveycollecting?nonce=483752766 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x100ec0aa0>: Failed to establish a new connection: [Errno 61] Connection refused'))
```

and errors regarding modules not found (in the docker terminal)
Are you able to add me on discord to continue debugging: thegodhand_

**Review by Julian-dev28 — COMMENTED** [review]
Looking good so far. Need to test scripts before approval.

File: network/core-node/admin-guide/monitoring.mdx
Comment (bboston7) [reply network/core-node/admin-guide/monitoring.mdx:190]: Just sent you a discord message

File: network/core-node/admin-guide/monitoring.mdx
Comment (bboston7) [reply network/core-node/admin-guide/monitoring.mdx]: I added links to the `scripts` directory and to the `OverlaySurvey.py` file.

**Review by anupsdf — APPROVED** [review]

File: network/core-node/admin-guide/monitoring.mdx
Comment (anupsdf) [inline network/core-node/admin-guide/monitoring.mdx:191]: I was able to run this script in Testnet and see the results.

**Pull request merged**
